osu.Game/Overlays/Chat/DrawableUsername.cs

```diff
@@ -156,6 +161,14 @@ public MenuItem[] ContextMenuItems
                 if (!user.Equals(api.LocalUser.Value))
                     items.Add(new OsuMenuItem(UsersStrings.CardSendMessage, MenuItemType.Standard, openUserChannel));
 
+                if (currentChannel != null)
+                {
+                    items.Add(new OsuMenuItem(ChatStrings.MentionUser, MenuItemType.Standard, () =>
+                    {
+                        currentChannel.Value.TextBoxMessage.Value += $"@{user.Username}";
```

Comment:
I think this should be
```suggestion
                        currentChannel.Value.TextBoxMessage.Value += $"@{user.Username} ";
```
for convenience. Rationale: Otherwise you have to make sure to type a space after the mention in order to not break the mention.

Apps like discord don't have this problem because they are presumably inserting some magic mention blocks rather than plain text, but as long as we're working with plain text for working mentions, it seems like a no-brainer.